### PR TITLE
modal: move common code to a new `isAnimated` method

### PR DESCRIPTION
**Pull request by GeoSot**

### CODE REVIEW

**Comment by mdo** [comment]
Does this fix a particular issue or is it in support of something else?

**Comment by GeoSot** [comment]
Not Something special. Tiding up same checks, that are written in different ways

File: js/src/modal.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ class Modal extends BaseComponent {
     this._ignoreBackdropClick = false
     this._isTransitioning = false
     this._scrollbarWidth = 0
+    this._isAnimated = element.classList.contains(CLASS_NAME_FADE)
```
[the remark this answers is not in the document]
Comment (GeoSot) [reply js/src/modal.js:93]: @rohit2sharma95 I suppose, no one changes html markup on runtime removing 'fade'. But either way, could be method

[the remark this answers is not in the document]
Comment (rohit2sharma95) [reply js/src/modal.js]: It does not make any breaking change though but changes the previous behavior IMO

**Review by rohit2sharma95 — APPROVED** [review]

**Pull request merged**